src/cdk/constructs/auto-delete-bucket/index.ts

```diff
@@ -10,7 +10,21 @@ import { Provider } from '@aws-cdk/custom-resources'
 import { Bucket, BucketProps } from '@aws-cdk/aws-s3'
 import { SingletonFunction, Runtime, Code } from '@aws-cdk/aws-lambda'
 
+/**
+ * A construct representing an AutoDeleteBucket
+ *
+ * @class AutoDeleteBucket
+ * @extends {Bucket}
+ * @category Construct
+ */
 export class AutoDeleteBucket extends Bucket {
+  /**
+   * Creates an instance of AutoDeleteBucket.
+   * @param {Construct} scope
+   * @param {string} id
+   * @param {BucketProps} [props={}]
+   * @memberof AutoDeleteBucket
```

Comment:
It seems wasteful to repeat these - do we not get any of these for free in the docs, given that we are using TS?

Reply:
I will remove these as they were leftover from a previous test (same as the policy aspect).